persistence/javadsl/src/test/scala/com/lightbend/lagom/javadsl/persistence/AbstractReadSideSpec.scala

```diff
@@ -37,46 +48,75 @@ trait AbstractReadSideSpec extends ImplicitSender { spec: ActorSystemSpec =>
 
   def getAppendCount(id: String): CompletionStage[java.lang.Long]
 
-  private def assertSelectCount(id: String, expected: Long): Unit = {
-    within(20.seconds) {
-      awaitAssert {
-        val count = Await.result(getAppendCount(id).toScala, 5.seconds)
-        count should ===(expected)
-      }
-    }
+  private val tag = TestEntity.Evt.AGGREGATE_EVENT_SHARDS.forEntityId("1")
```

Comment:
moved `tag` to instance val because it's also used on the `fetchLastOffset` method